consumer-configuration/README.adoc

```diff
@@ -0,0 +1,347 @@
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-version: 1
+:product-long: Application Services
+:product: App Services
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://cloud.redhat.com/beta/application-services/streams/
+:property-file-name: app-services.properties
+
+// Other upstream project names
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+
+//URL components for cross refs
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:getting-started-url: getting-started/README.adoc
+:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
+:kafkacat-url: kafkacat/README.adoc
+:quarkus-url: quarkus/README.adoc
+:rhoas-cli-url: rhoas-cli/README.adoc
+:rhoas-cli-ref-url: commands
+:topic-config-url: topic-configuration/README.adoc
+:consumer-config-url: consumer-configuration/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-configuring-topics"]
+= Configuring consumer groups in {product-long}
+ifdef::context[:parent-context: {context}]
+:context: configuring-consumer-groups
+
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+Consumer groups share large data streams generated by producers from a given topic in a Kafka instance.
+Grouping consumers scales consumption to keep up with the rate of data produced.
+Consumers within a group don’t read data from the same partition, but can receive data from one or more partitions.
```

Comment:
I thought that for a consumer to "read" and to "receive" data were the same, so to say consumers don't read from the same partition but receive data from one or more is kind of confusing/conflicting for me. Can you clarify what this means exactly?

Reply:
To avoid confusion, I've changed both to _read_